### PR TITLE
Link to StatProfileHTML in Profile docs

### PR DESCRIPTION
**Pull request by oxinabox**
If we are going to link to ProfileView then we need to be fair and do both.
I like StatProfileHTML better,

other option is to delete all reference

### CODE REVIEW

**Comment by KristofferC** [comment]
Also https://github.com/vchuravy/PProf.jl? :P

**Comment by ViralBShah** [comment]
Should we still get this in?

File: doc/src/manual/profile.md [hunk 1]
```diff
@@ -57,7 +57,8 @@ julia> using Profile
 julia> @profile myfunc()
 ```
 
-To see the profiling results, there is a [graphical browser](https://github.com/timholy/ProfileView.jl)
+To see the profiling results, there are several graphical browsers
+including [ProfileView.jl](https://github.com/timholy/ProfileView.jl) and [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML)
```
Comment (oxinabox) [inline doc/src/manual/profile.md:61]:
```suggestion
including [ProfileView.jl](https://github.com/timholy/ProfileView.jl), [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML), and [PProf.jl](https://github.com/vchuravy/PProf.jl).
```

**Comment by oxinabox** [comment]
yes, probably. because ProfileView has issues on Mac for GTK sometimes, and also handled very large profiles badly because it makes mega SVGs in Jupyter.

File: doc/src/manual/profile.md [hunk 2]
```diff
@@ -57,7 +57,8 @@ julia> using Profile
 julia> @profile myfunc()
 ```
 
-To see the profiling results, there is a [graphical browser](https://github.com/timholy/ProfileView.jl)
+To see the profiling results, there are several graphical browsers
+including [ProfileView.jl](https://github.com/timholy/ProfileView.jl), [https://github.com/tkluck/StatProfilerHTML.jl](StatProfilerHTML), and [PProf.jl](https://github.com/vchuravy/PProf.jl).
```
Comment (timholy) [inline doc/src/manual/profile.md:61]:
```suggestion
One "family" of visualizers is based on [FlameGraphs.jl](https://github.com/timholy/FlameGraphs.jl), with each family member providing a different user interface:
- [Juno](https://junolab.org/) is a full IDE with built-in support for profile visualization
- [ProfileView.jl](https://github.com/timholy/ProfileView.jl) is a stand-alone visualizer based on GTK
- [ProfileVega.jl](https://github.com/davidanthoff/ProfileVega.jl) uses VegaLight and integrates well with Jupyter notebooks
- [StatProfilerHTML](https://github.com/tkluck/StatProfilerHTML.jl) produces HTML and presents some additional summaries, and also integrates well with Jupyter notebooks
- [ProfileSVG](https://github.com/timholy/ProfileSVG.jl) renders SVG

An entirely independent approach to profile visualization is [PProf.jl](https://github.com/vchuravy/PProf.jl), which uses the external `pprof` tool.
```

Comment (timholy) [reply doc/src/manual/profile.md:61]: I wasn't even sure it's worth mentioning ProfileSVG, feel free to delete it if you want.

**Comment by oxinabox** [comment]
wow during the time since i wrote this it was utterly outdated i see.
Thanks @timholy

File: doc/src/manual/profile.md [hunk 3]
```diff
@@ -57,7 +57,15 @@ julia> using Profile
 julia> @profile myfunc()
 ```
 
-To see the profiling results, there is a [graphical browser](https://github.com/timholy/ProfileView.jl)
+To see the profiling results, there are several graphical browsers
```
Comment (timholy) [inline doc/src/manual/profile.md:60]:
```suggestion
To see the profiling results, there are several graphical browsers.
```

File: doc/src/manual/profile.md [hunk 4]
```diff
@@ -57,7 +57,15 @@ julia> using Profile
...
+One "family" of visualizers is based on [FlameGraphs.jl](https://github.com/timholy/FlameGraphs.jl), with each family member providing a different user interface:
+- [Juno](https://junolab.org/) is a full IDE with built-in support for profile visualization
+- [ProfileView.jl](https://github.com/timholy/ProfileView.jl) is a stand-alone visualizer based on GTK
+- [ProfileVega.jl](https://github.com/davidanthoff/ProfileVega.jl) uses VegaLight and integrates well with Jupyter notebooks
+- [StatProfilerHTML](https://github.com/tkluck/StatProfilerHTML.jl) produces HTML and presents some additional summaries, and also integrates well with Jupyter notebooks
+- [ProfileSVG](https://github.com/timholy/ProfileSVG.jl) renders SVG
+
+An entirely independent approach to profile visualization is [PProf.jl](https://github.com/vchuravy/PProf.jl), which uses the external `pprof` tool.
 available, but here we'll use the text-based display that comes with the standard library:
```
Comment (timholy) [inline doc/src/manual/profile.md:69]:
```suggestion

Here, though, we'll use the text-based display that comes with the standard library:
```

**Review by timholy — COMMENTED** [review]
Yeah, splitting ProfileView into LeftChildRightSiblingTree, FlameGraphs, ProfileView, and ProfileSVG led to a number of other opportunities.

**Comment by ViralBShah** [comment]
Would be nice to rebase, reword and get it merged!

**Comment by oxinabox** [comment]
Done, rebased and partially squashed

**Comment by KristofferC** [comment]
It should be noticed that this is GPL licensed.

**Comment by timholy** [comment]
Accidental cross post or something we need to deal with?

**Comment by KristofferC** [comment]
Nothing really that has to be dealt with but some people are a bit worried to get "tainted" by accidentally reading GPL code (derived work etc) like with StatProfileHTML. Don't know if it is a problem in practice.

**Comment by ViralBShah** [comment]
I think it is reasonable to expect that people will check the license of a piece of software before looking at it.

**Comment by timholy** [comment]
Thanks for explaining, I wasn't sure what you were referring to. Good catch. I don't think it matters for me (I don't plan on reading through its source), but it would be nice if GitHub displayed a "license summary" on the home screen.

**Pull request merged**